src/transformers/pipelines/feature_extraction.py

```diff
@@ -39,6 +41,9 @@ class FeatureExtractionPipeline(Pipeline):
             Device ordinal for CPU/GPU supports. Setting this to -1 will leverage CPU, a positive will run the model on
             the associated CUDA device id.
     """
+    def __init__(self, return_tensors=False, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.return_tensors = return_tensors
```

Comment:
This needs to be moved in `_sanitize_parameters` to be consistent with all arguments of pipelines.

Reply:
I have done everything you suggested. Test runs too.